tours/src/App.vue

```diff
@@ -1,57 +1,42 @@
 <template>
   <div v-if="toursAllowed" id="tours">
-    <oc-button
-      v-if="tours.length === 1"
-      id="toursButton"
-      v-oc-tooltip="tours[0].tooltip"
-      size="small"
-      @click.stop="startTour(0)"
-    >
+    <oc-button v-if="tours.length === 1" id="toursButton" v-oc-tooltip="tours[0].tooltip" size="small"
```

Comment:
Can you add a hover effect to the button? (fill the background and reverse the letters colour)

Reply:
Nvm, issue with the theme as well.